src/components/materialization/GenerateButton.tsx

```diff
@@ -161,50 +135,32 @@ function MaterializeTestButton({
                 ENTITY.MATERIALIZATION
             );
             if (draftSpecsResponse.error) {
-                return onFailure({
+                return callFailed({
                     error: {
                         title: 'materializationCreate.test.failure.errorTitle',
                         error: draftSpecsResponse.error,
                     },
                 });
             }
 
-            resetFormState(FormStatus.TESTING);
-            const publicationsSubscription = subscription(newDraftId);
-            const dryRunResponse = await createPublication(newDraftId, true);
-
-            if (dryRunResponse.error) {
-                return onFailure(
-                    {
-                        error: {
-                            title: `${messagePrefix}.test.failure.errorTitle`,
-                            error: dryRunResponse.error,
-                        },
-                    },
-                    publicationsSubscription
-                );
-            }
-
-            setDraftId(dryRunResponse.data[0].draft_id);
```

Comment:
Now the generate button just generates catalogs. So no need to run a dry run here